docs/developer-guide/ksqldb-reference/describe.md

```diff
@@ -95,3 +96,90 @@ messages-per-sec:      4.41   total-messages:       486     last-message: 12/14/
   failed-messages:         0      last-failed:       n/a
 (Statistics of the local ksqlDB server interaction with the Kafka topic IP_SUM)
 ```
+
+The following statement shows how to get the descriptions for all tables.
+
+```sql
+DESCRIBE TABLES;
+```
+
+Your output should resemble:
+
+```
+Name                 : IP_SUM
+ Field   | Type
+-----------------------------------------
+ IP      | VARCHAR(STRING)  (primary key)
+ KBYTES  | BIGINT
+-----------------------------------------
+For runtime statistics and query details run: DESCRIBE <Stream,Table> EXTENDED;
+Name                 : LO_REM
+
+ Field   | Type
+-----------------------------------------
+ LO      | VARCHAR(STRING)  (primary key)
+ KBYTES  | BIGINT
+-----------------------------------------
+For runtime statistics and query details run: DESCRIBE <Stream,Table> EXTENDED;
+```
+
+The following statement shows how to get the descriptions for all tables with 
+extended information.
+
+```sql
+DESCRIBE TABLES EXTENDED;
+```
+
+Your output should resemble:
+
+```
+Name                 : IP_SUM
+Type                 : TABLE
+Timestamp field      : Not set - using <ROWTIME>
+Key format           : KAFKA
+Value format         : JSON
+Kafka output topic   : IP_SUM (partitions: 4, replication: 1)
+
+ Field   | Type
+-----------------------------------------
+ IP      | VARCHAR(STRING)  (primary key)
+ KBYTES  | BIGINT
+-----------------------------------------
+
+Queries that write into this TABLE

```

Comment:
I'm not 100% sure if this is the same, it looks like this statement was moved to be under `statement` above

Reply:
I don't know -- please feel free to fix it as you see fit!